network-api/networkapi/wagtailpages/tests/buyersguide/test_article_page.py

```diff
@@ -206,3 +210,107 @@ def test_secondary_related_articles_no_related_articles(self):
         result = article_page.get_secondary_related_articles()
 
         self.assertListEqual(result, [])
+
```

Comment:
All the tests below are testing the form rather than the page type. What do you think about moving this into a separate class and potentially into a `test_forms.py` module, so that the test module structure matches the implementation module structure?

Reply:
This definitely makes sense to me, as we are indeed making changes to the form but not the model directly. 
The latest commit now includes `buyersguide/tests/test_forms.py`